pyomo/repn/tests/test_gams.py

```diff
@@ -94,6 +94,15 @@ def test_split_long_line(self):
             pat * 3477 + "var1 +\nlog(var2 / 9) - " +
             pat * 3043 + "x")
 
+    def test_solver_arg(self):
+        m = ConcreteModel()
+        m.x = Var()
+        m.c = Constraint(expr=m.x == 2)
+        m.o = Objective(expr=m.x)
+        os = StringIO()
+        m.write(os, format="gams", io_options=dict(solver="baron"))
+        self.assertIn("option lp=baron", os.getvalue())
```

Comment:
Can you change this to be "gurobi" instead of "baron"? Nick will tell you himself that Baron is not intended to solve LPs.